events/gameLeave.js

```diff
@@ -0,0 +1,29 @@
+function gameLeave(context, payload) {
+
+    // Get the game's ID
+    const {gameID} = payload;
+
+    // Get the game
+    const game = context.getGameState({gameID});
+
+    // Get the player's ID
+    const playerID = context.id();
+
+    // Filter away all players that aren't the one who is leaving
+    let otherPlayers = game.players.filter(function(player) {
```

Comment:
Do note that `game.players` is an object, not an array. To delete from an array you could use the `delete` keyword, that is: `delete game.players[playerId]`.